resources/standards-data/langtags/readme.md

```diff
@@ -0,0 +1,15 @@
+# langtags.json
+
+This file is retrieved from https://ldml.api.sil.org/langtags.json
+
+This file lists implicit script tags for language tags and equivalent BCP 47 tags per SLDR (and hence CLDR). We
+use this to build up good clean BCP 47 tags for adding languages to Windows.
+
+This file is intended to replace alltags, ethnologue, and iso639-3 data files in future. language-subtag-registry includes extra script metadata not present in langtags.json.
+
+## File Format
+
+* Source repository: https://github.com/silnrsi/langtags/
+* Documentation: https://github.com/silnrsi/langtags/blob/master/doc/langtags.md
+* Unofficial schema: https://gist.github.com/mcdurdin/eff65bb2d77034525120d48cbe280d30
```

Comment:
The attached langtags.json has API 1.2.1

Should the unofficial schema include the following?
iana [Always an array 1.2.0]
obsolete: boolean [Added 1.2.1]
unwritten: boolean [Added 1.2.1]

(These would be ignored in BuildLangTags.pas)

Reply:
Yes, the unofficial schema does need updating, but wanting to hand that over to Martin and have it added to the langtags repo.